.expeditor/scripts/shared.sh

```diff
@@ -33,3 +33,7 @@ curlbash_hab() {
     esac
     echo "--- :habicat: Hab binary set to $hab_binary"
 }
+
+get_toolchain() {
+  cat "rust-toolchain"
```

Comment:
This seems like it depends on the working directory of the caller to be the root of the repo. I'm not sure we can assume that. It might make it more robust to employ the source dir trick:
```bash
source_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```
And then access `rust-toolchain` via its relative path.

Reply:
It does depend on running from the root fo the repo, are there any situations where we wouldn't be? I'm taking into account Makefile based execution, buildkite based execution, and running manually.

Reply:
This path resolution will depend on the working directory of the user, regardless of where any of the code doing the sourcing lives. So, if a user were to manually run any script which sourced this function from a directory outside the repo or from a sub directory within the repo, it wouldn't work.

Reply:
Right, I understand that, but is that a valid use case that we want to support?

Reply:
I've run into breakages exactly like this elsewhere in our project, so yes, I would. It's not terribly hard to support and I'd say it makes the code more robust, so why not?